blocks/basic/include/gnuradio-4.0/basic/common_blocks.hpp

```diff
@@ -10,6 +10,22 @@
 #include <gnuradio-4.0/Graph.hpp>
 #include <gnuradio-4.0/reflection.hpp>
 
+// A convenience class to make writing unit tests easier.
+// This sink allows to inspect the input port values as a class member.
+template<typename T>
+class InspectSink : public gr::Block<InspectSink<T>> {
```

Comment:
This should be moved to gr::testing (and we should review the various test sinks at some point, there might be room for unification...)

Reply:
Done